src/System.Private.CoreLib/src/System/Collections/Generic/ArraySortHelper.cs

```diff
@@ -2,77 +2,37 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Could you please rename this to `ArraySortHelper.CoreCLR.cs` to follow the conventions ?